__tests__/url-verification.spec.js

```diff
@@ -0,0 +1,32 @@
+const verify = require('../src/url-verification');
+const fixture = require('../__fixtures__/url-verification.json');
+
+describe('URL verification', () => {
+  const OLD_ENV = process.env;
+
+  beforeEach(() => {
+    process.env = { ...OLD_ENV };
+  });
+
+  afterEach(() => {
+    process.env = OLD_ENV;
+  });
```

Comment:
Why is this needed when the `beforeEach` above exists?

Reply:
For consistency throughout the suite of tests. `afterEach` outside of this scope would have the updated values before they're reset, global cleanup would have process.env in an unexpected state if it needs it. It's also the only place you can conceivably reset it to the right values after the last test in this describe has run if you wanted/needed to.

Mostly for safety and completeness and given it's a tremendously cheap operation it's ok i figure.

Reply:
I believe each test is isolated so this wouldn't be an issue but not exactly impacting performance so :ok_hand:

Reply:
The tests are isolated per file by creating a worker pool of child processes that run tests but if you were to use --runInBand it uses a single process. Safety first, Guy. 👷

But yeah, mostly doesn't really matter

Reply:
🤦‍♂️